SQUIR-124 - Upgrade "where is the changelog" page (#1819)
akeneo/pim-docs: migrate_pim/changelog.rst

What is a Changelog?
========================

When our maintenance team works on issues, they take extreme care to avoid any backward compatibility break, but it might happen. 

> If it is the case, the BC break is mentioned in the changelog of the version.

Reviewer: ```suggestion
If it is the case, the backward compatibility break is mentioned in the changelog of the version.
```

Where is the Changelog?
========================

.. note::

    Akeneo provides 2 changelogs: one for CE edition available the Community repo and one for EE edition which is only available in the latest Enterprise Edition archive.

> Community Edition changelog :

Reviewer: ```suggestion
Community Edition changelog:
```

* Go to our `Community Repository <https://github.com/akeneo/pim-community-dev/>`_.
* Select your branch in the drop down menu, for instance select 6.0 branch.
* Then scroll down the page, and select the desired Changelog in the list, for instance `Changelog 6.0 version <https://github.com/akeneo/pim-community-dev/blob/6.0/CHANGELOG-6.0.md>`_.

> Enterprise Edition changelog :

Reviewer: ```suggestion
Enterprise Edition changelog:
```
